frappe/core/doctype/doctype/doctype.json

```diff
@@ -528,6 +530,15 @@
    "fieldname": "index_web_pages_for_search",
    "fieldtype": "Check",
    "label": "Index Web Pages for Search"
+  },
+  {
+   "fieldname": "default_email_template",
+   "fieldtype": "Data",
+   "label": "Default Email Template"
+  },
```

Comment:
The fieldtype should be Link and it should be linked to Email Template

Reply:
I don't think so. I continued the pattern set by Default Print Format, which also is not a link field. My guess is this prevents some errors when trying to delete a Print Format / Email Template.

Any comments from frappe on this matter?

Reply:
@barredterra Default Print Format is a Link field in Customize Form. In DocType it has always been a Data field from the start, not sure why.
Deletion of Linked Doctypes is already handled in Frappe, should not be an issue.
Let's make this a Link field in both places.

Reply:
@barredterra please resolve the comment above

Reply:
@marination, done (8732662). The behavior is as follows:

### Email Template Linked from Customize Form

- Delete **Email Template** -> works
- Write Email -> "undefined" in message, instead of template

### Email Template Linked from DocType

- Delete **Email Template** -> not possible because linked from **DocType**. **DocType** cannot be edited, so **Email Template** can never be deleted.